frontend/src/scenes/error-tracking/ErrorTrackingFilters.tsx

```diff
@@ -40,6 +40,38 @@ export const ErrorTrackingFilters = ({ showOrder = true }: { showOrder?: boolean
                                 setDateRange({ date_from: changedDateFrom, date_to: changedDateTo })
                             }}
                             size="small"
+                            showAbsoluteDatePickers={false}
+                            dateOptions={[
+                                {
+                                    key: 'Today',
+                                    values: ['dStart'],
+                                },
+                                {
+                                    key: 'Last 24 hours',
+                                    values: ['-24h'],
+                                },
+                                {
+                                    key: 'Last 7 days',
+                                    values: ['-7d'],
+                                },
+                                {
+                                    key: 'Last 14 days',
+                                    values: ['-14d'],
+                                },
+                                {
+                                    key: 'Last 30 days',
+                                    values: ['-30d'],
+                                },
+                                {
+                                    key: 'Last 90 days',
+                                    values: ['-90d'],
+                                },
+                                {
+                                    key: 'All time',
+                                    values: ['all'],
```

Comment:
What is a reasonable period to look over for "all" time? The sparkline labels are generated in advance so it would be pretty tough to predict what the time period of returned results will be unless we actually set it. Do we do anything similar elsewhere? Is two years grouped as months rational?

Reply:
i'd say maybe lets remove this, allow folk to set custom date ranges, and track what they query

my guess is you care about recent errors almost always
and probably only trends over time if you're looking for a long-running tricky problem

Reply:
I was avoiding the absolute dates because it's difficult to do them in the sparkline query

I've gone ahead and removed these, adding back in the custom dates. We just don't show the sparkline if you pick a defined period. Feels ok and I sense that will be an uncommon use case so people probably won't complain

Reply:
yep, we can always have a little info doohickie if folk are doing it a lot to let them know why its missing...
feels better than prompting people to choose a slow option 🙈